src/main/java/net/mcreator/ui/modgui/VillagerProfessionGUI.java

```diff
@@ -140,7 +142,16 @@ public VillagerProfessionGUI(MCreator mcreator, ModElement modElement, boolean e
 		displayName.setValidator(new TextFieldValidator(displayName,
 				L10N.t("elementgui.villager_profession.profession_needs_display_name")));
 		displayName.enableRealtimeValidation();
-		pointOfInterest.setValidator(new PointOfInterestValidator(mcreator, pointOfInterest, savedPointOfInterest));
+		pointOfInterest.setValidator(
+				new UniqueNameValidator("Point of interest", () -> pointOfInterest.getBlock().getUnmappedValue(),
```

Comment:
`"Point of interest"` should be externalized.

Reply:
Done